streams/early_zip_readable_streams_test.ts

```diff
@@ -60,3 +60,20 @@ Deno.test("earlyZipReadableStreams() can zip three streams", async () => {
     "3",
   ]);
 });
+
+Deno.test("earlyZipReadableStreams() forwards cancel()", async () => {
```

Comment:
Can you please use an assertion somewhere here?

Reply:
I used one in the cancel method

Reply:
Sorry, missed that.